### PR TITLE
feat: add config option for request timeout and handle the timeout in secure channel

### PR DESCRIPTION
**Pull request by alexbrdn**

### CODE REVIEW

File: uasc/secure_channel.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func NewSecureChannel(endpoint string, c *uacp.Conn, cfg *Config) (*SecureChanne
 		reqhdr: &ua.RequestHeader{
 			AuthenticationToken: ua.NewTwoByteNodeID(0),
 			Timestamp:           time.Now(),
-			TimeoutHint:         0xffff,
+			TimeoutHint:         uint32(cfg.RequestTimeout.Round(time.Millisecond).Nanoseconds() / 1e6),
```
Comment (magiconair) [inline uasc/secure_channel.go]: I think this should be just
```go
uint32(cfg.RequestTimeout/time.Millisecond)
```

**Review by magiconair — COMMENTED** [review]
Small improvement. Could you also add an `Option` like this one, please?

https://github.com/gopcua/opcua/blob/master/config.go#L46-L53

File: uasc/secure_channel.go (hunk 1 above)
Comment (alexbrdn) [reply uasc/secure_channel.go]: Absolutely. That was stupid of me :)

**Comment by alexbrdn** [comment]
added `Option` and cleaned up as suggested

File: uasc/secure_channel.go (hunk 1 above)
Comment (magiconair) [reply uasc/secure_channel.go]: nah :)

**Comment by magiconair** [comment]
LGTM

**Pull request merged**